### DISCUSSION TITLE
ANDROID

### DISCUSSION THREAD

**Discussion by hamadachande94** [community/community#54889] [Programming Help]
### Select Topic Area

Show & Tell

### Body

WHICH WAYS THAT LEARNERS CAN USE TO LEARN ANDROID STUDIO IN SIMPLE WAYS?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @hamadachande94 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by iamreiyn** [comment]
Hi @hamadachande94 

There are several ways that you can use to learn Android Studio in simple ways...Here are some suggestions:

1. Online tutorials- There are many online tutorials available that can help you to learn Android Studio. These tutorials range from video courses to written tutorials that explain the basics of Android Studio.

2. Documentation: Android Studio documentation provides a comprehensive guide for beginners to learn the basics of Android Studio. It is an excellent resource if you prefer to learn through reading and reference.

3. Practice projects: Creating simple projects in Android Studio can help you understand the various features of the software. Projects can be created from scratch or by following tutorials.

4. Community forums: You can use online forums like Stack Overflow or even this Community discussion page and ask questions to get answers or more information about a particular thing. You can also participate as it will also help you increase your knowledge about the Android Studio...

5. YouTube videos: YouTube has a vast collection of videos that teach the basics of Android Studio. You can watch these videos to visualize the software in action and understand its various features...

6. Online courses: There are several online courses available that teach Android Studio. These courses are structured and provide a step-by-step guide to learning Android Studio...

So these are some of the ways you could start your Android Studio learning journey, I hope it helped you, thank you!

**Reply by hamadachande94** [reply]
I have tried much to push work on GitHub while are fail while in uploading are successfully 

C:\Users\BINCHANDE>cd desktop

C:\Users\BINCHANDE\Desktop>cd regularexp

C:\Users\BINCHANDE\Desktop\regularexp>git init
Initialized empty Git repository in C:/Users/BINCHANDE/Desktop/regularexp/.git/

C:\Users\BINCHANDE\Desktop\regularexp>git add README.md
fatal: pathspec 'README.md' did not match any files

C:\Users\BINCHANDE\Desktop\regularexp>git config --global user.name "hamadachande94"

C:\Users\BINCHANDE\Desktop\regularexp>git config --global user.email "binchande94@gmail.com"

C:\Users\BINCHANDE\Desktop\regularexp>git add .

C:\Users\BINCHANDE\Desktop\regularexp>git commit -m "first commit"
[master (root-commit) 1e4e8e6] first commit
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 REGULAR EXPRESSION BY CHANDE.pdf

C:\Users\BINCHANDE\Desktop\regularexp>git remote add origin https://github.com/hamadachande94/bita521054.git

C:\Users\BINCHANDE\Desktop\regularexp>git push -u origin master
Username for 'https://github.com':
Password for 'https://github.com':
remote: No anonymous write access.
fatal: Authentication failed for 'https://github.com/hamadachande94/bita521054.git/'

C:\Users\BINCHANDE\Desktop\regularexp>git push -u origin master
Username for 'https://github.com': hamadachande94
Password for 'https://hamadachande94@github.com':
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/hamadachande94/bita521054.git/'

**Comment by hamadachande94** [comment]
thanks very much

**Reply by hamadachande94** [reply]
than I need this number to make in regular expression
bita/2/22/000/tz
bita222000tz
